### PR TITLE
Execute the sidebar action with Ctrl+Shift+Comma

### LINKED ISSUE #392: Sidebar keyboard Shortcut
I'd love it if I could toggle the Bitwarden sidebar using a keyboard combo, similar to `Ctrl+Shift+U` (on Linux) to show the popup.

If you're open to this idea, I'd be glad to submit a PR.

### PR DESCRIPTION
**Pull request by cadecairos**
Fixes #392 

I'm not sure if this works or not, and I'd like to test this myself, but the extension is not able to build on Linux..

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -106,6 +106,11 @@
         "default": "Ctrl+Shift+Y",
         "linux": "Ctrl+Shift+U"
       }
+    },
+    "_execute_sidebar_action": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+Comma"
```
Comment (kspearrin) [inline src/manifest.json]: Have we cleared this shortcut as being available on all platforms?

Comment (cadecairos) [reply src/manifest.json]: I don't see it listed on this page: https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly (but we should double check 😃)

Comment (cadecairos) [reply src/manifest.json]: I wonder if it makes more sense to have this shortcut more closely match the browser action shortcut, but use the `Alt` modifier instead:

```
"_execute_sidebar_action": {
  "suggested_key": {
    "default": "Alt+Shift+Y",
    "linux": "Alt+Shift+U"
  }
}
```

Comment (kspearrin) [reply src/manifest.json]: @cadecairos I like the `Alt` idea much better. Let's do that. I just checked that shortcut page on Windows (it changes based on your user agent) and `Alt+Shift+Y` is available.

Comment (cadecairos) [reply src/manifest.json]: Awesome, I've pushed up a new commit making that change.

**Comment by kspearrin** [comment]
Thanks

**Pull request merged**